snapshot/lib/snapshot/fixes/simulator_zoom_fix.rb

```diff
@@ -7,8 +7,6 @@ module Fixes
 
     class SimulatorZoomFix
       def self.patch
-        Snapshot.kill_simulator # First we need to kill the simulator
```

Comment:
It doesn't seem necessary to kill simulators so many times for each iteration of the test invocation loop. We'll do this once, apply all fixes, and then run the tests.